diff/windows/windows.go

```diff
@@ -320,10 +320,13 @@ func mountsToLayerAndParents(mounts []mount.Mount) (string, []string, error) {
 		return "", nil, fmt.Errorf("number of mounts should always be 1 for Windows layers: %w", errdefs.ErrInvalidArgument)
 	}
 	mnt := mounts[0]
-	if mnt.Type != "windows-layer" {
+
+	if mnt.Type != "windows-layer" && mnt.Type != "bind" {
 		// This is a special case error. When this is received the diff service
 		// will attempt the next differ in the chain which for Windows is the
 		// lcow differ that we want.
+		// TODO: Is there any situation where we actually wanted a "bind" mount to
+		// fall through to the lcow differ?
```

Comment:
afair, this was just to handle unpacking linux images from the `Apply` path. lcow has a type of lcow-layer so we should be fine

Reply:
Yeah, if it's a generic feature, then I think the current approach of "Explicitly allow-list mount types we understand, and let others fall through to whatever can handle it" is better. Technically, we probably should reject an upper layer `bind` mount the same way, right now it'll come though as 'Invalid Argument' (no parents) and fail rather than try the next differ. (I _think_ that logically a parentless layer will always be a bind mount and vice versa here by dint of the snapshotter implementation, but am not immediately certain that outside forces won't break that assumption.)